packages/components/src/placeholder/style.scss

```diff
@@ -54,6 +54,11 @@
 	}
 }
 
+.components-placeholder__fieldset.is-column-layout,
+.components-placeholder__fieldset.is-column-layout form {
```

Comment:
Any thoughts on overriding this style in `packages/block-library/src/table/editor.scss` as opposed to introducing a new prop? Will the prop be useful in any other blocks?

Reply:
Not sure. The existing blocks all have a horizontal layout, but so far there are no others with multiple inputs. That is probably the scenario where the column layout is most useful. Is it likely there will be more of these in the future?

Reply:
I'd lean towards _You Ain't Gonna Need It_, but no strong preference either way 🙂

Reply:
Hmm good point, I'll look into a table-specific override.

Reply:
OK so there's no way of doing this without changing the Placeholder component in some way, because the CSS for column layout needs to be added to an inner element, not to the wrapper that takes the `className` prop. The alternative to `isColumnLayout` would be to pass in a second className prop, say fieldsetClassName, to be applied to the inner div. That seems a bit messier, and less self-explanatory, so will stick with the current solution.

Reply:
We could pass in `className="wp-block-table__placeholder"` and then have:

```css
.wp-block-table__placeholder .components-placeholder__fieldset,
.wp-block-table__placeholder .components-placeholder__fieldset form,
{
	flex-direction: column;
}
```

But, yes, as this point it's not very tidy. I'll defer to your judgement! 🙂

Reply:
I've been using CSS Modules for too long, keep forgetting that nesting classes is a thing 😭